Cargo.toml

```diff
@@ -21,6 +22,8 @@ env_logger = "0.9.1"
 futures = "0.3.17"
 # TODO: Replace this with crate.io published version
 in-toto = { git = "https://github.com/Xynnn007/in-toto-rs.git", rev = "7f69799" }
+# TODO: Replace this with crate.io published version
+kbs-types = { git = "https://github.com/sameo/kbs-types.git", branch = "topic/kbs" }
```

Comment:
I will switch to the upstream branch once https://github.com/virtee/kbs-types/pull/11 is merged.

Reply:
The kbs-types PR is merged now, and I updated that PR to point to upstream.